package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "zerosum",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "babel-core": "^6.26.3",
```

Comment:
A lot of these are `devDependencies`.

Reply:
I'm not sure which specifically are devDependencies...  I'm guessing webpack and babel dependencies?  Or would all front end dependencies be devDependencies?